doc/account.md

```diff
@@ -0,0 +1,35 @@
+Account
+=======
+
+The `account` options in `config.json` allow you to set how long an account is viable at times during it's lifespan. There are current only settings for `initiate` and `complete`. We want the lifetime for an initial account creation to be short so we can clean up accounts which haven't been completed and won't have numerous files sitting in storage. Also we set a `complete` account to 6 months; to comply with PCI standards.
+
+The object assigned to each account section is passed into the OtDate object to set the expires for each lifetime.
+
+    "account": {
+        "completeLifetime": {
+            "months": 6
+        },
+        "initiateLifetime": {
+            "hours": 1
+        }
+    }
+
+
+Lifespan
```

Comment:
I don't see "lifespan" in the object.  I also don't see a goo description of what the lifespan is.  I think I know what you're trying to get at, so I'd suggest adding more words in the paragraph on line 21.